src/include/execution/executable_query.h

```diff
@@ -58,5 +100,13 @@ class ExecutableQuery {
   // together.
   std::unique_ptr<util::Region> region_;
   std::unique_ptr<ast::Context> ast_ctx_;
+  std::unique_ptr<brain::OperatingUnits> operating_units_;
+
+  // Used to specify the output for this query
+  std::unique_ptr<exec::SampleOutput> sample_output_;
+  std::unique_ptr<exec::OutputPrinter> printer_;
```

Comment:
These test outputs and printers actually should not be put here... **I (Lin)** will move them out since I actually wrote this part. Just remember to merge this part back when I'm done.

Reply:
I've done moving this. Please merge my branch in.